lang/english.lng

```diff
@@ -1,2 +1,2 @@
 ##grflangid 0x01
-STR_GENERAL_DESC        :OpenSFX sound replacement set for OpenTTD. Freely available under the terms of these licenses: Creative Commons Attribution-ShareAlike 3.0 Unported, GNU General Public License version 2 (or later) and Common Development and Distribution License 1.1. For full credits see "readme.txt". [{TITLE}]
+STR_GENERAL_DESC        :OpenSFX sound replacement set for OpenTTD. Freely available under the terms of these licenses: CC BY-SA 3.0, GPLv2 (or later) and CDDL 1.1. For full credits see "readme.txt". [{TITLE}]

```

Comment:
If we're being picky, this should be "licences" (en-GB), with the en-US version as "licenses" (as you have it).